src/test/java/emissary/server/mvc/ShutdownActionTest.java

```diff
@@ -1,13 +1,13 @@
 package emissary.server.mvc;
 
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class ShutdownActionTest extends EndpointTestBase {
+class ShutdownActionTest extends EndpointTestBase {
```

Comment:
Should either delete this placeholder file or create a follow-on ticket to implement tests

Reply:
Ticket created